pubsublite/spark-connector/README.md

```diff
@@ -0,0 +1,184 @@
+# Using Spark SQL Streaming with Pub/Sub Lite
+
+The samples in this directory show how to read messages from and write messages to Pub/Sub Lite from an [Apache Spark] cluster using the [Pub/Sub Lite Spark Connector].
```

Comment:
The instruction is specifically target at Dataproc, I think you should say it explicitly here rather than any spark cluster (since other Spark Cluster might offer different versions/deps package)

Reply:
Updated.